lib/util/event.js

```diff
@@ -1,10 +1,24 @@
+const fs    = require('fs');
+const homedir = require('os').homedir();
+const path = require('path');
 const EventSource = require('EventSource');
 const { HOST } = require('./api');
 
 module.exports = {
   listen: (endpoint) => {
     return new Promise((resolve, reject) => {
-      const eventSource = new EventSource(HOST + endpoint);
+      const configPath = path.join(homedir, '.mothership-cli');
+      const apiTokenPath = path.join(configPath, 'apiToken');
+      let apiToken;
+
+      if (fs.existsSync(configPath) && fs.existsSync(apiTokenPath)) {
+        apiToken = fs.readFileSync(path.join(configPath, 'apiToken'));
+      }
+
+      const eventSource = new EventSource(
+        HOST + endpoint,
+        { headers: { 'Authorization': `Bearer ${apiToken}` } }
+      );
```

Comment:
Kinda minor but since you already retrieve `apiToken` inside of `/util/api.js` you could just expose it as an export of that file and use it here like this:

```js
...
{ headers: { 'Authorization': `Bearer ${api.API_TOKEN}` } }
...
```